### PR TITLE
PG-788: Font applied in settings should be applied in the export file

### PR DESCRIPTION
**Pull request by phillip-hopper**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/glyssen/226)

### CODE REVIEW

**Comment by tombogle** [comment]
I don't think we should apply the vernacular font to the localized character ID. (Ultimately, we might well need to have a place/way that the font family and size can be specified for each localized language, but we can't safely assume it should be the same as the vernacular.)
We *do* need to use the font info from the reference text(s) for those columns.

File: Glyssen/ProjectExporter.cs [hunk 1]
```diff
@@ -357,18 +358,34 @@ private void GenerateExcelFile(string path, IEnumerable<List<object>> data)
 				sheet.Column(columnNum++).AutoFit(2d, sheet.DefaultColWidth); // verse
 				sheet.Column(columnNum++).AutoFit(2d, 20d); // character ID
 
-				// add a column for the localized character id
```
Comment (tombogle) [inline Glyssen/ProjectExporter.cs]: I don't think we want to remove this code. If we really don't want to auto-fit this column (and why not?), it doesn't feel like it has anything to do with this Jira issue. In any case, I think we would need to increment columnNum.

Comment (phillip-hopper) [inline Glyssen/ProjectExporter.cs]: Undone.

File: Glyssen/ReferenceText.cs [hunk 2]
```diff
@@ -533,5 +533,15 @@ protected override string ProjectFolder
 				return m_projectFolder;
 			}
 		}
+
```
Comment (tombogle) [inline Glyssen/ReferenceText.cs]: Note: This has already been added (elsewhere in the file) in master, so this will result in a build error when merged.

**Pull request merged**